### DISCUSSION TITLE
Un user que cuenta con los permisos correctos en el workflow no ve ningún grupo

### DISCUSSION THREAD

**Discussion by nicolasdurans** [Cotalker/documentation#743] [4. Bug Reports]
![image](https://user-images.githubusercontent.com/5656833/157462279-2569245c-d481-4a8f-beee-2cd4b35b8b23.jpeg)

Me ha pasado cuando en la compañía cotalker al agregar un partner cómo user (jose luis Bizama o Giovanny como ejemplo) le agrego todos los permisos respectivos para que se vea el workflow y no logra verse solo se ven los canales que se activan

**Comment by acedward** [comment]
Si nos puedes enviar información de como reproducir el problema, o las credenciales (vía email) podríamos ver este problema. En general con los permisos respectivos deberían poder entrar al taskgroup respectivo
